src/pages/special/panel/AdminPanel.jsx

```diff
@@ -14,10 +19,33 @@ const AdminPanel = () => {
       <h2>
         Welcome
         <span style={{ color: "red" }}>
-          {auth?.currentUser?.displayName ?? "Saro-crew"}
+          {auth?.currentUser?.displayName ?? " Saro-crew"}
         </span>
       </h2>
-      <p>Admin main page</p>
+      {isLoading && <DefaultLoader />}
+      <CardGroup>
+        {nav &&
+          nav.map(({ title, subcontent, path }, index) => {
+            return subcontent ? (
+              <Card key={index}>
+                <Card.Body>
+                  <Card.Title className="title">{title}</Card.Title>
+                  {subcontent.map(({ title, path }) => {
```

Comment:
{subcontent.map(({ title, path }) => (<Link to={...{path, key: path}}>{title}</Link>;
                  ))} - tak wygląda czytelniej, co? :)